src/ProductBundle/Resources/views/Product/view.html.twig

```diff
@@ -69,10 +69,16 @@ file that was distributed with this source code.
                             {% block product_generic_properties %}
                                 <li class="list-group-item">
                                     {% block product_unit_price %}
-                                        <small class="list-group-item-text">
-                                            {{ 'product_price'|trans({}, "SonataProductBundle") }}:&nbsp;
-                                            <span itemprop="price">{{ sonata_product_price(product, currency, true)|number_format_currency(currency) }}</span>
-                                        </small><br/>
+                                        {% if product.enabled %}
+                                            <small class="list-group-item-text">
+                                                {{ 'product_price'|trans({}, "SonataProductBundle") }}:&nbsp;
+                                                <span itemprop="price">{{ sonata_product_price(product, currency, true)|number_format_currency(currency) }}</span>
+                                            </small><br/>
+                                        {% else %}
+                                            <small class="list-group-item-text">
+                                                <span>{{ 'product_not_available'|trans({}, "SonataProductBundle") }}</span>
+                                            </small><br/>
+                                        {% endif %}
```

Comment:
The outer `small` tag does not change, you should put it outside of the condition.